src/subscriber.ts

```diff
@@ -94,36 +81,51 @@ export class Message {
      * @name Message#ackId
      * @type {string}
      */
-    this.ackId = ackId;
+    this.ackId = ackId!;
```

Comment:
Why is the `!` needed here?

Reply:
I switched to use the generated types, before I just hand wrote it but I don't think we had a solution in place for that yet. As such, they mark everything as optional (even though that isn't necessarily the case)